contracts/VALORTimedMint.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/access/Ownable.sol"; //not 100% necessary I think, just a modifier would suffice as well
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract VALORTimedMint is ERC20, Ownable {
+
+    uint256 public timeUntilNextMint;
+    uint256 public claimPeriod;
+    uint256 private mintCap;
+
+    event Claim(address claimant, uint256 amount);
+    
+    constructor(
+        uint256 _mintCap,
+        uint256 _timeDelay,
+        uint256 mintAmount
+    )   
+        ERC20("VALORIZEDAO", "VALOR") 
+    {
+        timeUntilNextMint = block.timestamp + _timeDelay;
+        claimPeriod = timeUntilNextMint + 60 days; //subject to change
+        mintCap = _mintCap;
+        _mint(address(this), mintAmount);
+    }
+
+    function mint(address destination, uint256 amount) external onlyOwner {
```

Comment:
Let's make this function internal and remove the modifier.  That will be handled in the child contract.

Reply:
I would actually remove this function altogether and move all the logic to the `_mint()` function on line 46.